src/components/FileTreeNode/index.tsx

```diff
@@ -8,7 +8,9 @@ import { gettext } from '../../utils';
 import { TreeNode } from '../FileTree';
 import styles from './styles.module.scss';
 
-type PublicProps = TreefoldRenderProps<TreeNode>;
+type PublicProps = TreefoldRenderProps<TreeNode> & {
+  onSelect: (id: string) => void;
```

Comment:
The type for this same function (although the argument is named `id` here and `path` above) is also declared in `FileTree/index.tsx` (at line 129). I wonder if it would be worth defining its own type (e.g., `TreeNodeOnSelect`), that could be used in both places?

Reply:
That's a good idea.